### ISSUE TITLE
Does a Higher `fasttext_oh_eli5_vs_rw_v2_prob` Indicate Better Data Quality?

### ISSUE THREAD

**Issue by huyiwen** [mlfoundations/dclm#53]
Thanks for the excellent work! I have a question: Does a higher `fasttext_oh_eli5_vs_rw_v2_prob` score indicate better model performance, or is a lower score preferable?

**Comment by afang-story** [comment]
Higher is considered preferable or more likely to be "higher quality" data but we don't make any particularly strong statements and there is definitely interesting future work to be done regarding what it means to be "high quality."
